packages/react-select/src/internal/index.js

```diff
@@ -5,3 +5,4 @@ export { default as DummyInput } from './DummyInput';
 export { default as NodeResolver } from './NodeResolver';
 export { default as ScrollBlock } from './ScrollBlock';
 export { default as ScrollCaptor } from './ScrollCaptor';
```

Comment:
Can we remove these components in this PR since they've been replaced by `ScrollManager`?